src/data/markdown/docs/03 cloud/04 Integrations/06 Grafana Plugin.md

```diff
@@ -3,10 +3,10 @@ title: 'Grafana Plugin'
 excerpt: 'View your k6 Cloud Test Results in Grafana'
 ---
 
-The [k6 Cloud application plugin](https://grafana.com/grafana/plugins/grafana-k6-app/) for [Grafana](https://grafana.com/grafana/) allows you to:
-* manage and visualize cloud tests to make your testing accessible in Grafana.
+The [k6 Cloud app](https://grafana.com/grafana/plugins/grafana-k6-app/) for [Grafana](https://grafana.com/) makes your testing accessible with Grafana, allowing you to:
```

Comment:
@e-fisher 

On the doc, it always names it as `k6 Cloud app`, so I did an attempt to be consistent on the intro.

Still unsure. Let me know if this is inaccurate or exists a better alternative to discuss it.

Reply:
Yeah, that looks better, thank you!